### DISCUSSION TITLE
Application Log shows received data but KNX Device does nothing

### DISCUSSION THREAD

**Discussion by RSBlek** [Supergiovane/node-red-contrib-knx-ultimate#350] [General]
Hello supergiovane, Hello community,

i am new to to knx + node red and can't bring my knx-ultimate to receive data from my KNX IP Router.

**Describe the bug**
The "KNX Device" Node does not output any received group address values. The physical KNX Device sends a temperature and humidty value every Minute on Group Addresses 1/0/0 and 1/1/0. I can see in the Node-Red Application Log, that it is receiving Messages on this Group Addresses. But the Node doeos not output any of this received values. I will add screenshots of my full configuration. I would be very happy if someone could help me to find my error. I tried everything possible for hours. But couldn't find my mistake. Thanks a lot!!

**To Reproduce**
I have a really simple setup. Clean Node-Red Installation, MDT IP Router, One Device in the Bus that sends a value every minute.

**Expected behavior**
I saw videos, where the "KNX Device" Node instantly shows the received value under the node. And i expect to see the received value in a debug-node that i connected to the knx device.

**Screenshots**
Screenshot of ETS Group Monitor
![2024-04-21 16_50_52-ETS6 - Home](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/38309568/5ad1cd91-a6d1-4d77-baea-2188b8fd1ca7)

Screenshot of knxUltimate-config
![2024-04-21 16_52_22-Node-RED _ Flow 1](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/38309568/09770232-905d-401b-827b-90f5a24f5a09)

Screenshot of my KNX Device Node
![2024-04-21 16_53_47-Node-RED _ Flow 1](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/38309568/53afb991-b5dd-4eb4-8f31-64389d85662f)

Screenshot of my Node Red Flow
![2024-04-21 16_55_01-](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/38309568/92602c5d-d1c4-47c6-995c-583c8f2dde8e)

**Knx-Ultimate Version**
2.4.22

**Are you running node-red behind homematic, docker or anything similar?**
Tried in Node Red on Docker and Windows

**Additional context**
I also tried the universal mode. But the result was the same. No output.

Log Output on Startup
```
[info] 21.4.2024, 16:44:46.88 knxUltimate-config: perform websocket connection on KNX Gateway
[info] 21.4.2024, 16:44:46.88 KNXUltimate-config: Connecting... KNX Gateway
[info] 21.4.2024, 16:44:46.89 knxUltimate-config: Connected to {
  ipAddr: '224.0.23.12',
  ipPort: 3671,
  physAddr: KNXAddress { type: 0, level: 3, _address: 65302, length: 2 },
  suppress_ack_ldatareq: false,
  loglevel: 'trace',
  localEchoInTunneling: true,
  hostProtocol: 'Multicast',
  isSecureKNXEnabled: false,
  jKNXSecureKeyring: null,
  localIPAddress: '192.168.178.26',
  interface: 'WLAN',
  connectionKeepAliveTimeout: 30,
  localSocketAddress: undefined
}
[trace] 21.4.2024, 16:44:49.310 KNXUltimate-KNXEngine: Received KNX packet: _processInboundMessage, Data received: 0610020b00160801c0a8b21ad1e20804010208060700 srcAddress: {"address":"192.168.178.26","family":"IPv4","port":53730,"size":22} ChannelID:null
[trace] 21.4.2024, 16:44:49.311 KNXUltimate-KNXEngine: Received KNX packet: _processInboundMessage, Data received: 06100201000e0801c0a8b21ad1e2 srcAddress: {"address":"192.168.178.26","family":"IPv4","port":53730,"size":14} ChannelID:null
[warn] 21.4.2024, 16:44:52.97 KNXUltimate-config: unable to read peristent file \Users\abl\.node-red\knxultimatestorage\knxpersistvalues\knxpersista457813901cc23a7.json ENOENT: no such file or directory, open '\Users\abl\.node-red\knxultimatestorage\knxpersistvalues\knxpersista457813901cc23a7.json'
```

Application Log while Node-Red is running. You can see the received packets on 1/0/0 and 1/1/0
```
[trace] 21.4.2024, 16:48:00.286 KNXUltimate-KNXEngine: Received KNX packet: _processInboundMessage, Data received: 0610020b00160801c0a8b21ad1e20804010208060700 srcAddress: {"address":"192.168.178.26","family":"IPv4","port":53730,"size":22} ChannelID:null
[trace] 21.4.2024, 16:48:00.287 KNXUltimate-KNXEngine: Received KNX packet: _processInboundMessage, Data received: 06100201000e0801c0a8b21ad1e2 srcAddress: {"address":"192.168.178.26","family":"IPv4","port":53730,"size":14} ChannelID:null
[trace] 21.4.2024, 16:48:07.349 KNXUltimate-KNXEngine: Received KNX packet: _processInboundMessage, Data received: 0610020b00160801c0a8b21ad1e20804010208060700 srcAddress: {"address":"192.168.178.26","family":"IPv4","port":53730,"size":22} ChannelID:null
[trace] 21.4.2024, 16:48:07.350 KNXUltimate-KNXEngine: Received KNX packet: _processInboundMessage, Data received: 06100201000e0801c0a8b21ad1e2 srcAddress: {"address":"192.168.178.26","family":"IPv4","port":53730,"size":14} ChannelID:null
[trace] 21.4.2024, 16:48:14.425 KNXUltimate-KNXEngine: Received KNX packet: _processInboundMessage, Data received: 0610020b00160801c0a8b21ad1e20804010208060700 srcAddress: {"address":"192.168.178.26","family":"IPv4","port":53730,"size":22} ChannelID:null
[trace] 21.4.2024, 16:48:14.426 KNXUltimate-KNXEngine: Received KNX packet: _processInboundMessage, Data received: 06100201000e0801c0a8b21ad1e2 srcAddress: {"address":"192.168.178.26","family":"IPv4","port":53730,"size":14} ChannelID:null
[trace] 21.4.2024, 16:48:19.54 KNXUltimate-KNXEngine: Received KNX packet: _processInboundMessage, Data received: 0610053000202900bcd0110308001003f110002e0e17222eb90207af1b855ea2 srcAddress: {"address":"192.168.178.63","family":"IPv4","port":3671,"size":32} ChannelID:null
[debug] 21.4.2024, 16:48:19.57 KNXUltimate-KNXEngine: Received KNX packet: ROUTING: L_DATA_IND, Data: {"_tpci":3,"_apci":241,"_data":{"_data":{"type":"Buffer","data":[16,0,46,14,23,34,46,185,2,7,175,27,133,94,162]}}} srcAddress: 1.1.3 dstAddress: 1/0/0 Host:224.0.23.12:3671
[trace] 21.4.2024, 16:48:19.139 KNXUltimate-KNXEngine: Received KNX packet: _processInboundMessage, Data received: 0610053000202900bcd0110309001003f110002e0e17222f7332d82f3268424c srcAddress: {"address":"192.168.178.63","family":"IPv4","port":3671,"size":32} ChannelID:null
[debug] 21.4.2024, 16:48:19.140 KNXUltimate-KNXEngine: Received KNX packet: ROUTING: L_DATA_IND, Data: {"_tpci":3,"_apci":241,"_data":{"_data":{"type":"Buffer","data":[16,0,46,14,23,34,47,115,50,216,47,50,104,66,76]}}} srcAddress: 1.1.3 dstAddress: 1/1/0 Host:224.0.23.12:3671
[trace] 21.4.2024, 16:48:21.491 KNXUltimate-KNXEngine: Received KNX packet: _processInboundMessage, Data received: 0610020b00160801c0a8b21ad1e20804010208060700 srcAddress: {"address":"192.168.178.26","family":"IPv4","port":53730,"size":22} ChannelID:null
[trace] 21.4.2024, 16:48:21.491 KNXUltimate-KNXEngine: Received KNX packet: _processInboundMessage, Data received: 06100201000e0801c0a8b21ad1e2 srcAddress: {"address":"192.168.178.26","family":"IPv4","port":53730,"size":14} ChannelID:null
```

**Comment by Supergiovane** [comment]
Hi
Is the WLAN your only interface?
Binding to a wireless interface is not recommended, because you could lose packets.

**Comment by Supergiovane** [comment]
Enable the "universal mode". You should see all telegrams, not only the 1/0/0.

**Comment by RSBlek** [comment]
Hi, i also tried the universal mode and connecting my computer directly to the switch, where my KNX IP Router is connected.
The result is unfortunately the same: The received packets are visible in the node red application log, but the "KNX Device" Node does not output anything.

Since it seems to be a problem that only occurs in my environment and is not a general problem. I will have to debug it even more or use a other library.

Thanks!
